src/content/docs/serverless-function-monitoring/aws-lambda-monitoring/enable-lambda-monitoring/containerized-images.mdx

```diff
@@ -0,0 +1,74 @@
+---
+title: Instrument your containerized image layer
+metaDescription: A guide for instrumenting your containerized image layer with New Relic.
+freshnessValidatedDate: never
+---
+
+If you're using a containerized image for Lambda, you'll want to add New Relic to your code. After you add our pre-built layer to your image, you'll need to push the image to your AWS Elastic Container Registry. From there, whenever you run your function, New Relic will run, too. 
+
+Here's a guide to adding our pre-built layer to your code. 
+
+<Steps>
+<Step>
+### Add New Relic to your Dockerfile:
+New Relic offers pre-built layers for different runtimes: Java, Node.js, and Python.
+
+1. Select the layer compatible with your specific Lambda function's runtime and region:
+ * [Java](https://gallery.ecr.aws/x6n7b2o2/newrelic-lambda-layers-java)
+ * [Node.js](https://gallery.ecr.aws/x6n7b2o2/newrelic-lambda-layers-nodejs) 
+ * [Python](https://gallery.ecr.aws/x6n7b2o2/newrelic-lambda-layers-python)
+3. **At the top of your Dockerfile** add the following code: 
+
+ <Tabs>
+ <TabsBar>
+ <TabsBarItem id="1">Dockerfile for Java</TabsBarItem>
+ <TabsBarItem id="2">Dockerfile for Node.js</TabsBarItem>
+ <TabsBarItem id="3">Dockerfile for Python</TabsBarItem>
```

Comment:
We should add in option 4 Dockerfile for Extension

Reply:
Good call!